diff --git i/asyn/Makefile w/asyn/Makefile
index 3f803f48..20974179 100644
--- i/asyn/Makefile
+++ w/asyn/Makefile
@@ -15,7 +15,7 @@ ASYN = $(TOP)/asyn
 USR_CFLAGS += -DUSE_TYPED_RSET -DUSE_TYPED_DSET -DUSE_TYPED_DRVET
 USR_CPPFLAGS += -DUSE_TYPED_RSET -DUSE_TYPED_DSET -DUSE_TYPED_DRVET
 
-USR_INCLUDES_cygwin32 += -I/usr/include/tirpc
+USR_INCLUDES_cygwin32 += `pkg-config --cflags libtirpc`
 
 # The following gets rid of the -fno-implicit-templates flag on vxWorks, 
 # so we get automatic template instantiation.
@@ -33,7 +33,7 @@ asyn_SYS_LIBS_cygwin32 = $(CYGWIN_RPC_LIB)
 # Some linux systems moved RPC related symbols to libtirpc
 # Define TIRPC in configure/CONFIG_SITE in this case
 ifeq ($(TIRPC),YES)
-  USR_INCLUDES_Linux += -I/usr/include/tirpc
+  USR_INCLUDES_Linux += `pkg-config --cflags libtirpc`
   asyn_SYS_LIBS_Linux += tirpc
 endif
 
diff --git i/testGpibApp/src/Makefile w/testGpibApp/src/Makefile
index 20957003..e619dc28 100644
--- i/testGpibApp/src/Makefile
+++ w/testGpibApp/src/Makefile
@@ -43,7 +43,7 @@ testGpibVx_SRCS_vxWorks += testGpibVx_registerRecordDeviceDriver.cpp
 testGpib_LIBS += devTestGpib
 testGpib_LIBS += testSupport asyn
 ifeq ($(TIRPC),YES)
-  USR_INCLUDES += -I/usr/include/tirpc
+  USR_INCLUDES += `pkg-config --cflags libtirpc`
   testGpib_SYS_LIBS += tirpc
 endif
 SYS_PROD_LIBS_cygwin32 += $(CYGWIN_RPC_LIB)
