diff --git a/src/tools/EPICS/Release.pm b/src/tools/EPICS/Release.pm
index 7caa6f4c6..67347eb45 100644
--- a/src/tools/EPICS/Release.pm
+++ b/src/tools/EPICS/Release.pm
@@ -108,8 +108,8 @@ sub expandMacros {
     # $Rmacros is a reference to a hash
 
     while (my ($pre, $var, $post) = $str =~ m/ (.*) \$\( ($MVAR) \) (.*) /x) {
-        last unless exists $Rmacros->{$var};
-        $str = $pre . $Rmacros->{$var} . $post;
+        last unless (exists $Rmacros->{$var} or exists $ENV{$var});
+        $str = $pre . ($Rmacros->{$var} or $ENV{$var}) . $post;
     }
     return $str;
 }
