diff --git a/src/ua_types.c b/src/ua_types.c
index b8f6739bb..7e8f7c43d 100644
--- a/src/ua_types.c
+++ b/src/ua_types.c
@@ -1880,7 +1880,7 @@ UA_Array_delete(void *p, size_t size, const UA_DataType *type) {
 }
 
 #ifdef UA_ENABLE_TYPEDESCRIPTION
-UA_Boolean
+UA_Boolean UA_EXPORT
 UA_DataType_getStructMember(const UA_DataType *type, const char *memberName,
                             size_t *outOffset, const UA_DataType **outMemberType,
                             UA_Boolean *outIsArray) {
