diff --git i/asyn/Makefile w/asyn/Makefile
index 1f2c21c7..e0aecbbb 100644
--- i/asyn/Makefile
+++ w/asyn/Makefile
@@ -33,7 +33,7 @@ asyn_SYS_LIBS_cygwin32 = $(CYGWIN_RPC_LIB)
 # Some linux systems moved RPC related symbols to libtirpc
 # Define TIRPC in configure/CONFIG_SITE in this case
 ifeq ($(TIRPC),YES)
-  USR_INCLUDES_Linux += -I$(SYSROOT)/usr/include/tirpc
+  USR_INCLUDES += $(shell pkg-config --cflags libtirpc)
   asyn_SYS_LIBS_Linux += tirpc
 endif
 
diff --git i/testGpibApp/src/Makefile w/testGpibApp/src/Makefile
index fe806643..4b7abc0b 100644
--- i/testGpibApp/src/Makefile
+++ w/testGpibApp/src/Makefile
@@ -43,7 +43,7 @@ testGpibVx_SRCS_vxWorks += testGpibVx_registerRecordDeviceDriver.cpp
 testGpib_LIBS += devTestGpib
 testGpib_LIBS += testSupport asyn
 ifeq ($(TIRPC),YES)
-  USR_INCLUDES += -I$(SYSROOT)/usr/include/tirpc
+  USR_INCLUDES += $(shell pkg-config --cflags libtirpc)
   testGpib_SYS_LIBS += tirpc
 endif
 SYS_PROD_LIBS_cygwin32 += $(CYGWIN_RPC_LIB)
