diff --git a/asyn/Makefile b/asyn/Makefile
index 7ce295c8..4774be2b 100644
--- a/asyn/Makefile
+++ b/asyn/Makefile
@@ -33,7 +33,7 @@ asyn_SYS_LIBS_cygwin32 = $(CYGWIN_RPC_LIB)
 # Some linux systems moved RPC related symbols to libtirpc
 # Define TIRPC in configure/CONFIG_SITE in this case
 ifeq ($(TIRPC),YES)
-  USR_INCLUDES_Linux += -I/usr/include/tirpc
+  USR_INCLUDES += `pkg-config --cflags libtirpc`
   asyn_SYS_LIBS_Linux += tirpc
 endif
 
diff --git a/testGpibApp/src/Makefile b/testGpibApp/src/Makefile
index 20957003..e619dc28 100644
--- a/testGpibApp/src/Makefile
+++ b/testGpibApp/src/Makefile
@@ -43,7 +43,7 @@ testGpibVx_SRCS_vxWorks += testGpibVx_registerRecordDeviceDriver.cpp
 testGpib_LIBS += devTestGpib
 testGpib_LIBS += testSupport asyn
 ifeq ($(TIRPC),YES)
-  USR_INCLUDES += -I/usr/include/tirpc
+  USR_INCLUDES += `pkg-config --cflags libtirpc`
   testGpib_SYS_LIBS += tirpc
 endif
 SYS_PROD_LIBS_cygwin32 += $(CYGWIN_RPC_LIB)
