diff --git i/build.gradle.kts w/build.gradle.kts
index 5297c1cf..2af6f040 100644
--- i/build.gradle.kts
+++ w/build.gradle.kts
@@ -446,7 +446,10 @@ tasks.register<War>("mgmtWar") {
 		include("*.sql")
 		into("install")
 	}
-	from(project.projectDir.resolve("docs/build")) { into("ui/help") }
+	from(project.projectDir.resolve("docs/build")) {
+		exclude("**/.doctrees")
+		into("ui/help")
+	}
 	from(project.projectDir.resolve("docs/source/samples")) {
 		include("deployMultipleTomcats.py")
 		into("install")
