diff --git i/build.gradle.kts w/build.gradle.kts
index a1075be4..e97a5ffc 100644
--- i/build.gradle.kts
+++ w/build.gradle.kts
@@ -405,7 +405,7 @@ tasks.register<War>("retrievalWar") { // Corrected typo from retreivalWar
 tasks.register<Tar>("buildRelease") {
 	group = "Wars"
 	description = "Builds a full release and zips up in a tar file."
-	dependsOn("mgmtWar", "retrievalWar", "etlWar", "engineWar", "generateReleaseNotes")
+	dependsOn("mgmtWar", "retrievalWar", "etlWar", "engineWar")
 
 	archiveFileName.set("archappl_v${version}.tar.gz")
 	compression = Compression.GZIP
